pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -758,7 +758,7 @@ func (v *vpnSeedServer) deployScrapeConfig(ctx context.Context) error {
 	)
 
 	if v.values.HighAvailabilityEnabled {
-		jobName, serviceNameRegexSuffix = "openvpn-server-exporter", "[0-2]"
+		jobName, serviceNameRegexSuffix = "openvpn-server-exporter", "-[0,1]"
```

Comment:
Great catch to find the missing `-`.

Please leave the range as is, though. @MartinWeindel intentionally made it bigger so that it would be easier to have a larger `StatefulSet` if required.

@MartinWeindel WDYT?